### DISCUSSION TITLE
search for words with hyphen

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1236] [Q&A]
Is it possible to search work like "web-hosting" ?
I am asking because web.hosting is returning also.

type "web-hosting" &&  -"web.hosting" is an option?

tks

**Comment by lfcnassif** [comment]
It is, but index options should be changed before processing the case, take a look at `extraCharsToIndex` option at conf/IndexTaskConfig.txt in 4.0 or conf/AdvancedConfig.txt in 3.18, you can add - as an extra char to be indexed.

But be careful, searches for e.g. `john` will stop to return `john-smith@gmail.com`, then you will have to search for `john*`

**Reply by paulobreim** [reply]
Hummm thank you. Better use as is. 

Tks
